src/bio/bio_buffer.c

```diff
@@ -709,11 +709,17 @@ dma_rw(struct bio_desc *biod, bool prep)
 
 			if (biod->bd_update)
 				spdk_blob_io_write(blob, channel, payload,
-						   pg_idx, pg_cnt,
+						   page2io_unit(biod->bd_ctxt,
+							   	pg_idx),
+						   page2io_unit(biod->bd_ctxt,
+							   	pg_cnt),
```

Comment:
(style)  code indent should use tabs where possible
(style)  please, no space before tabs